gossip/c_block_callbacks.go

```diff
@@ -53,6 +53,11 @@ var (
 	blockExecutionTimer = metrics.GetOrRegisterTimer("chain/execution", nil)
 	blockWriteTimer     = metrics.GetOrRegisterTimer("chain/write", nil)
 	blockAgeGauge       = metrics.GetOrRegisterGauge("chain/block/age", nil)
+
+	_ = metrics.GetOrRegisterMeter("chain/reorg/executes", nil)
+	_ = metrics.GetOrRegisterMeter("chain/reorg/add", nil)
+	_ = metrics.GetOrRegisterMeter("chain/reorg/drop", nil)
+	_ = metrics.GetOrRegisterMeter("chain/reorg/invalidTx", nil)
```

Comment:
@rus-alex what is the point in adding metrics, which will be unused?

Reply:
I agree we should add as much metrics from geth as we can, however if we cannot fill the metric with any data and we create it unused, it seems useless to me - is there some usecase which needs them to be present, even through they will have no meaningful value?

Reply:
It just method to say: we didn't forget implemented such metrics and they better than geth's.
Agreed, it is overmach.